docs/guides/admin/docs/workflowoperationhandlers/cut-marks-to-smil-woh.md

```diff
@@ -0,0 +1,66 @@
+CutMarksToSmilWorkflowOperationHandler
+===================================
+
+Description
+-----------
+
+This operation parses a JSON containing cut marks into a SMIL that can be used by the 
+[VideoEditorWorkflowOperation](editor-woh.md). It does this by attributing the given times to the tracks in the 
+given presentation and presenter flavors. 
+
+## Parameter Table
+
+|configuration keys         |example                |description                                                    |
+|------------------         |-----------            |---------------------------------------------------------------|
+|source-presenter-flavor    |`presenter/prepared`   |The flavor of the presenter video track. Must contain exactly one file.                   |
+|source-presentation-flavor |`presentation/prepared`|The flavor of the presentation video track. Must contain exactly one file.                  |
+|source-smil-flavor         |`smil/times`           |The flavor of the JSON. Must contain exactly one file.|
```

Comment:
Naming this configuration key `source-smil-flavor` is confusing. Based on this name I would expect a smil file, which is xml. However, you expect a JSON with the cut marks from BBB. Therefore I suggest to name it `source-json-flavor` and to amend the description with the information that these are the cut marks from BBB.

Reply:
Yup, that makes sense.
I would like to keep BBB out of the description, because I wouldn't want people to think that this WHO can only be used with BBB.